tests/cypress/integration/reportRepair/address.spec.js

```diff
@@ -1,43 +1,67 @@
 import {
   intercept_address_search,
   navigateToAddressPage,
-  navigateToPostcodePage
+  navigateToPostcodePage,
+  navigateToNotEligiblePageWhenPostcodeNotFound
 } from '../../support/helpers';
 
-function setup_addresses_search(setup_addresses_API) {
+function setup_addresses_search(setup_addresses_API, numberOfResults = 2) {
   setup_addresses_API();
-  navigateToAddressPage();
+  if (numberOfResults >= 1) {
+    navigateToAddressPage()
+  } else {
+    navigateToNotEligiblePageWhenPostcodeNotFound();
+  }
 }
 
 describe('address', () => {
   context('Content', () => {
     before(() => setup_addresses_search(intercept_address_search));
 
-    it('displays the label', () => {
-      cy.contains('Select an address');
+    it('displays the caption', () => {
+      cy.contains('Select the address');
     });
 
     it('button displays correct text', () => {
       cy.get('button').contains('Continue');
     });
 
-    it('contains a can\t find my address link', () => {
-      cy.contains('I can\'t find my address').click();
+    it('postcode change link shown', () => {
+      cy.contains('Change');
+    });
+
+    
+
+    it('contains a my address is not listed link', () => {
+      cy.contains('My address is not listed').click();
       cy.url().should('include', '/report-repair/not-eligible');
     });
   });
 
+  context('Content no Addresses returned', () => {
```

Comment:
`Content: no Addresses returned`?

Reply:
I've changed this to
`Content with no Addresses returned`

For consistency since the other context in the file is just 'Content'